CMakeLists.txt

```diff
@@ -97,12 +97,11 @@ set_target_properties(${APP_NAME} PROPERTIES
 )
 
 target_link_libraries(${APP_NAME}
+    "-framework ${BANUBA_OEP_FRAMEWORK}"
+    bnb_effect_player
     "-framework Accelerate"
     "-framework Cocoa"
     "-framework Foundation"
-    "-framework OpenGL"
-    "-framework BanubaEffectPlayer"
```

Comment:
The main framework banuba_sdk, must it be included, or does it work differently in the scene?